### PR TITLE
Updated Rádio Animu Presence

### PR DESCRIPTION
**Pull request by bossatossa**
Just changing to the new logo, adding support to a new page. ~~and adding a button that redirects to the website when the stream is playing.~~

Discord_UTraQkGxKd

### CODE REVIEW

**Comment by fr0r** [comment]
Just letting the reviewers know that @Bas950 is aware of the changes.

image

File: websites/R/Rádio Animu/presence.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 const browsingStamp = Math.floor(Date.now() / 1000);
-let artist;
-let title;
-let playing;
+let artist: string;
+let title: string;
+let playing: boolean ;
 
-presence.on("iFrameData", (data) => {
+presence.on("iFrameData", (data: any) => {
```
Comment (ririxidev) [inline websites/R/Rádio Animu/presence.ts:13]: No ``any`` type.

**Review by ririxidev — CHANGES_REQUESTED** [review]
@bossatossa,
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/R/Rádio Animu/presence.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ presence.on("UpdateData", async () => {
     presenceData.state = title;
     presenceData.smallImageKey = "play";
     presenceData.smallImageText = (await strings).play;
+    presenceData.buttons = [{label: "Ouça aqui!", url: "https://www.animu.com.br/"}];
```
[the remark this answers is not in the document]
Comment (bossatossa) [reply websites/R/Rádio Animu/presence.ts]: oops, my bad!
fixed.

File: websites/R/Rádio Animu/presence.ts (hunk 1 above)
Comment (bossatossa) [reply websites/R/Rádio Animu/presence.ts]: fixed

File: websites/R/Rádio Animu/dist/metadata.json [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
   },
   "contributors": [
     {
       "name": "fror",
       "id": "515668127829458945"
-    }
+    },
+	{
+	 "name": "Bas950",
+     "id": "241278257335500811"
```
Comment (EncryptedDev) [inline websites/R/Rádio Animu/dist/metadata.json:13]: fix indentation (prettier)

File: websites/R/Rádio Animu/presence.ts [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
     play: "presence.playback.playing"
   });
 
 const browsingStamp = Math.floor(Date.now() / 1000);
-let artist;
-let title;
-let playing;
+let artist: string;
+let title: string;
+let playing: boolean ;
```
Comment (EncryptedDev) [inline websites/R/Rádio Animu/presence.ts:11]: Merge your lets

File: websites/R/Rádio Animu/dist/metadata.json [hunk 5]
```diff
@@ -1,27 +1,33 @@
...
+  "version": "1.2.6",
+  "logo": "https://i.imgur.com/SHAH0WR.png",
+  "thumbnail": "https://i.imgur.com/YS3pllQ.png",
   "color": "#2c004b",
   "tags": [
     "brasil",
-    "anime"
+    "anime",
+	"radio",
+	"moe"
```
Comment (EncryptedDev) [inline websites/R/Rádio Animu/dist/metadata.json:30]: fix the indentation here too

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hi, look at my review. I'd just suggest you to prettify your code (:

File: websites/R/Rádio Animu/dist/metadata.json (hunk 5 above)
Comment (bossatossa) [reply websites/R/Rádio Animu/dist/metadata.json]: thanks for the suggestions 👍🏼 
applied them on https://github.com/PreMiD/Presences/pull/3430/commits/999f86f0c9d30287ba49744aa6aaf7a12d89e700

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by ririxidev — APPROVED** [review]
LGTM!!!!!!!!!!

**Pull request merged**